### DISCUSSION TITLE
Subdomains Issue

### DISCUSSION THREAD

**Discussion by akash9676** [vercel/community#6668] [Help]
### Summary

I took a domain in HostGator and i created some subdomains and hosted websites under subdomains in HostGator 

I created a website in nextjs and deployed in vercel and i have added subdomain in vercel, It took a day to update the subdomain URL
but i worked eventually

Now after my subdomain in vercel starts working all the domain and subdomains in HostGator in are not working how can i solve this issue

### Steps to Reproduce

![image](https://github.com/vercel/community/assets/75379553/7c671b0c-2b7f-4063-a86c-4505f969ee21) subdomain not working which is in HostGator

![image](https://github.com/vercel/community/assets/75379553/56acec5a-e627-41fa-86a9-3048ce59542d) subdomain working which is in vercel

**Comment by versecafe** [comment]
@akash9676 can you share how you configured the domains in Vercel, if you configured the cname records for *.domain.com then it would break all other subdomains being used from outside Vercel
